Tema-Java8/src/main/java/java8/Java8Main.java

```diff
@@ -0,0 +1,278 @@
+package java8;
+
+import javax.swing.text.html.Option;
+import java.awt.*;
+import java.util.*;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Java8Main {
+    //I made it 'global'. that's the only way I know to use it in 2 or more methods :(
+    public static Set<String> randomString = new HashSet<>();
```

Comment:
you can also declare it a class level and reuse it in difference functions / methods